VirtoCommerce.Platform.Web/Web.config

```diff
@@ -112,22 +113,43 @@
         </connectionManagement>
     </system.net>
 
+    <cacheManager.Redis>
+        <connections>
+            <connection id="redisConnectionString" database="0" connectionString="SECRET" />
+        </connections>
+    </cacheManager.Redis>
+
     <cacheManager>
         <managers>
-            <cache name="platformCache" updateMode="None" enableStatistics="false" enablePerformanceCounters="false">
+            <cache name="defaultPlatformCache" updateMode="None" enableStatistics="false" enablePerformanceCounters="false">
+                <handle name="memCacheHandle" ref="memCacheHandle" expirationMode="Sliding" timeout="10m" />
+            </cache>
+            <cache name="redisPlatformCache" enableStatistics="true" backplaneName="redisConnectionString" backplaneType="VirtoCommerce.Platform.Web.Cache.RedisCacheBackplane2, VirtoCommerce.Platform.Web">
                 <handle name="memCacheHandle" ref="memCacheHandle" expirationMode="Sliding" timeout="10m" />
+                <handle name="redisConnectionString" ref="redisHandle" isBackplaneSource="true" />
             </cache>
         </managers>
         <cacheHandles>
+            <handleDef id="redisHandle" type="VirtoCommerce.Platform.Web.Cache.RedisCacheHandle2`1, VirtoCommerce.Platform.Web" />
             <handleDef id="memCacheHandle" type="CacheManager.SystemRuntimeCaching.MemoryCacheHandle`1, CacheManager.SystemRuntimeCaching" />
         </cacheHandles>
+
+        <!--<managers>
+            <cache name="platformCache" updateMode="None" enableStatistics="false" enablePerformanceCounters="false">
+                <handle name="memCacheHandle" ref="memCacheHandle" expirationMode="Sliding" timeout="10m" />
+            </cache>
+        </managers>
+        <cacheHandles>
+            <handleDef id="memCacheHandle" type="CacheManager.SystemRuntimeCaching.MemoryCacheHandle`1, CacheManager.SystemRuntimeCaching" />
+        </cacheHandles>-->
     </cacheManager>
     <system.runtime.caching>
         <memoryCache>
             <namedCaches>
                 <!--For some cases, this setting physicalMemoryLimitPercentage = "memory usage threshold in percents",
                 leads to a complete stop of the cache working even after the current physical memory usage returns less that set's threshold--> 
-                <add name="memCacheHandle" />
+                <!--<add name="memCacheHandle" />-->
+                <add name="memCacheHandle" physicalMemoryLimitPercentage="80" pollingInterval="00:00:30" />
```

Comment:
this line also need to be removed

Reply:
Should it be removed actually? Are these parameters useless?